### PR TITLE
Feature/groups UI

### PR DESCRIPTION
**Pull request by Voxanimus**

### CODE REVIEW

File: app/src/main/java/com/github/multimatum_team/multimatum/adaptater/UserGroupAdapter.kt
Comment (felko) [inline app/src/main/java/com/github/multimatum_team/multimatum/adaptater/UserGroupAdapter.kt:28]: It seems that you need to notify the adapter that the data has changed using `notifyDataSetChanged()` just after updating `dataSource`.

File: app/src/main/java/com/github/multimatum_team/multimatum/adaptater/UserGroupAdapter.kt
Comment (felko) [inline app/src/main/java/com/github/multimatum_team/multimatum/adaptater/UserGroupAdapter.kt:15]: Imports not used

File: app/src/main/res/layout/activity_groups.xml
Comment (felko) [inline app/src/main/res/layout/activity_groups.xml:1]: Would you mind adding a bit of padding around the group list?

**Review by felko — CHANGES_REQUESTED** [review]

**Review by felko — APPROVED** [review]
Looks good, thanks for helping me on the groups feature.
Please wait for my PR to be merged before merging yours.

File: app/src/test/java/com/github/multimatum_team/multimatum/GroupActivityTest.kt
Comment (LennyDZ) [inline app/src/test/java/com/github/multimatum_team/multimatum/GroupActivityTest.kt:4]: Maybe add a todo tag, to be sure you'll do these test

**Review by LennyDZ — APPROVED** [review]
There is for sure way too many file. I tried to spot big mistakes, but it's too large to be exhaustive.

**Pull request merged**
